src/scripting/scriptingService.ts

```diff
@@ -8,16 +8,24 @@ import ConnectionManager from '../controllers/connectionManager';
 import { ScriptingRequest, ScriptingParams, ScriptOperation, ScriptingObject, ScriptOptions } from '../models/contracts/scripting/scriptingRequest';
 import { TreeNodeInfo } from '../objectExplorer/treeNodeInfo';
 import VscodeWrapper from '../controllers/vscodeWrapper';
+import { MetadataService } from '../metadata/metadataService';
+import { IConnectionCredentials } from '../models/interfaces';
+import { ObjectMetadata, MetadataType } from '../models/contracts/metadata/metadataRequest';
+import Utils = require('../models/utils');
```

Comment:
Any particular reason you're using require here? It's not wrong per-se but generally we should try to use import when we can.